### ISSUE TITLE
Bug: some posts are cut off when rotating from landscape to portrait

### ISSUE THREAD

**Issue by andrewbaker-uk** [Dimillian/IceCubesApp#1656]
## Environment:

 - OS: iPadOS 17.1
 - IceCubesApp version: 1.9.6

## Description

When rotating the iPad Pro from landscape to portrait some posts are cut off, because the side pane is on top of the post. This only happens to some posts.

![image](https://github.com/Dimillian/IceCubesApp/assets/10560135/c5884405-c5b3-4c28-9348-30fb80c84874)

## Related Issues

 - [X] Search that this bugs don't already exist before creating it.

**Comment by andrewbaker-uk** [comment]
Closing the app and reopening it does resolve the issue. But there may be a issue

**Comment by Dimillian** [comment]
Yep known issue with the image. You can also scroll them outside of the view and scroll back to them it should fix it.
